sdk/eventhub/Azure.Messaging.EventHubs/src/Processor/ProcessEventArgs.cs

```diff
@@ -21,8 +21,8 @@ public struct ProcessEventArgs
     {
         /// <summary>
         ///   Indicates whether or not the arguments contain an event to be processed.  In
-        ///   the case where no event is contained, then the context and creation of
-        ///   checkpoints are also unavailable.
+        ///   the case where no event is contained, then and creation of checkpoints and reading the last
```

Comment:
this sounds strange to me. Did you mean this?
```suggestion
        ///   the case where no event is contained, then the creation of checkpoints and reading the last
```

Reply:
yes, yes I did...